x/params/legacy/v044/store.go

```diff
@@ -0,0 +1,20 @@
+package v043
+
+import (
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+func migrateParamsStore(ctx sdk.Context, paramstore paramtypes.Subspace) {
+	paramstore.WithKeyTable(paramtypes.ConsensusParamsKeyTable())
```

Comment:
When Sunny did it like this for the staking param store, we ended up finding a bug when doing migrations.

I believe we need something like https://github.com/cosmos/cosmos-sdk/pull/9484